tools/build_framework.sh

```diff
@@ -79,7 +79,7 @@ case "$PUBLISH_STEP" in
         echo "---"
         echo "Build complete, skipping publish step."
         echo "Use one of the following additional arguments to get something that runs on a cluster:"
-        echo "- 'local': Host the build in a local HTTP server for use by a vagrant cluster."
+        echo "- 'local': Host the build in a local HTTP server for use by a DC/OS Vagrant cluster."
```

Comment:
Ah, okay. This might be confusing with both just `Vagrant` and `DC/OS Vagrant`. Would something like `local DC/OS cluster` work instead.